server/models/user.js

```diff
@@ -0,0 +1,14 @@
+
+export default (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    email: DataTypes.STRING,
```

Comment:
I will like you to add two fields `username` and `password` to the `user` model. Make the username unique and ensure that both fields don't accept null values.
 
Please note that when you make a change in the models, you have to make a corresponding change in the migration files and then rerun your migrations so that your DB will be up to date.

Reply:
Okay i would do that